### DISCUSSION TITLE
1.17.1 Fabric

### DISCUSSION THREAD

**Discussion by XStarlight** [Kron4ek/minecraft-vortex-launcher#26] [Q&A] [answered]
Hello. Still doesn't work when i try to launch fabric profile after setup fabric.

**Comment by Kron4ek** [comment, marked as answer]
Hello. 

Do you have `Download missing libraries on game start` enabled in the launcher settings? If not, enable it and try to launch the game again.

**Reply by XStarlight** [reply]
I just have enabled and still nothing happens after launch.

**Reply by Kron4ek** [reply]
Ok, actually it doesn't work indeed, i was able to reproduce the issue. I will upload a new launcher version a little later.

**Reply by XStarlight** [reply]
Thank you very much :)
The best launcher for testing mods and playing solo btw. Very appreciate!

**Reply by Kron4ek** [reply]
I uploaded a new version, please check if Fabric works now.

**Reply by XStarlight** [reply]
Yep, it works now, thanks.

**Comment by Dargus777** [comment]
I only have version 1.12.2 of forge, you can't download the version with forge in the launcher, you can only download the regular version without forge there, how do I download version 1.16.5 of forge?

**Reply by Kron4ek** [reply]
You can download Forge installers from [here](https://files.minecraftforge.net/net/minecraftforge/forge/index_1.16.5.html).

**Comment by Dargus777** [comment]
what should I do if there is no multilingual in minecraft?

**Reply by Kron4ek** [reply]
I don't know, try to redownload the game.
